organizations/organizations.go

```diff
@@ -246,3 +246,40 @@ func UpdateLogo(w http.ResponseWriter, r *http.Request) {
 
 	utils.GetSuccess("organization logo updated successfully", nil, w)
 }
+
+func UpdateSettings(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	orgId := mux.Vars(r)["id"]
+	requestData := make(map[string]interface{})
+	if err := utils.ParseJsonFromRequest(r, &requestData); err != nil {
+		utils.GetError(err, http.StatusUnprocessableEntity, w)
+		return
+	}
+
+	org_filter := make(map[string]interface{})
+	// org_filter["settings"] = requestData["settings"]
+
+	for key, element := range requestData {
+		org_filter[key] = element
+    }
+
+	fmt.Println("filter: ", org_filter)
+
+	update, err := utils.UpdateOrCreateOneMongoDbDoc(OrganizationSettings, orgId, org_filter)
```

Comment:
Lets stick with the update we have, no need to introduce a new function. We are doing an update for the settings field of an organization in the organization collection. This would mean that we add a settings filed to the organization struct since it holds the schema of an organization in the collection.